plugins/woocommerce/tests/e2e-pw/tests/smoke-tests/update-woocommerce.spec.js

```diff
@@ -46,6 +46,9 @@ const getWCDownloadURL = async () => {
 		headers: {
 			Accept: 'application/vnd.github+json',
 		},
+		params: {
+			per_page: 100
+		}
```

Comment:
Added this to allow us to select much older versions. The default `per_page` is `30`, which will not allow us to choose WC versions `7.0.0` and older (this E2E test would fail because it couldn't find `7.0.0` in the list). Expanding `per_page` to 100 will allow us to do so.